OpenRA.Game/FieldLoader.cs

```diff
@@ -467,202 +479,345 @@ public static object GetValue(string fieldName, Type fieldType, MiniYaml yaml, M
+				case "HotkeyReference":
+					return ParseHotkeyReference(fieldName, value, fieldType);
+				case "WDist":
+					return ParseWDistance(fieldName, value, fieldType);
+				case "WVec":
+					return ParseWVector(fieldName, value, fieldType);
+				case "WVec[]":
+					return ParseWVectors(fieldName, value, fieldType);
+				case "WPos":
+					return ParseWPosition(fieldName, value, fieldType);
+				case "WAngle":
+					return ParseWAngle(fieldName, value, fieldType);
+				case "WRot":
+					return ParseWRot(fieldName, value, fieldType);
+				case "CPos":
+					return ParseCPosition(fieldName, value, fieldType);
+				case "CVec":
+					return ParseCVector(fieldName, value, fieldType);
+				case "CVec[]":
+					return ParseCVectors(fieldName, value, fieldType);
+				case "BooleanExpression":
+					return ParseBoolExpression(fieldName, value, fieldType);
+				case "IntegerExpression":
+					return ParseIntegerExpression(fieldName, value, fieldType);
+				case "ImageFormat":
+					return ParseImageFormat(fieldName, value, fieldType);
+				case "Boolean":
+					return ParseBool(fieldName, value, fieldType);
+				case "Int32[]":
+				case "Int64[]":
+				case "String[]":
+				case "HSLColor[]":
+				case "Boolean[]":
+				case "ImageFormat[]":
+				case "IntegerExpression[]":
+				case "BooleanExpression[]":
+				case "CPos[]":
+				case "WRot[]":
+				case "WAngle[]":
+				case "WPos[]":
+				case "WDist[]":
+				case "HotkeyReference[]":
+				case "Hotkey[]":
+				case "BlinkState[]":
+				case "Single[]":
+					return ParseArray(fieldName, value, fieldType, field);
+				case "HashSet`1":
+					return ParseHashSet(fieldName, value, fieldType, field);
+				case "Dictionary`2":
+					return ParseDict(fieldName, value, fieldType, yaml, field);
+				case "Size":
+					return ParseSize(fieldName, value, fieldType);
+				case "int2":
+					return ParsePairInt(fieldName, value, fieldType);
+				case "float2":
+					return ParsePairFloat(fieldName, value, fieldType);
+				case "float3":
+					return ParseTripletFloat(fieldName, value, fieldType);
+				case "Rectangle":
+					return ParseRectangle(fieldName, value, fieldType);
+				case "DateTime":
+					return ParseDateTime(fieldName, value, fieldType);
+				case "WindowMode":
+				case "TextCAlign":
+				case "ClientState":
+				case "TextAlign":
+				case "TextVAlign":
+				case "MapGridType":
+				case "MapVisibility":
+				case "VisibilityType":
+				case "DecisionMetric":
+				case "ExplosionType":
+				case "Stance":
+				case "PipType":
+				case "ReferencePoints":
+				case "DamageState":
+				case "PowerState":
+				case "OwnerType":
+				case "Unit":
+				case "UncloackType":
+				case "UnitStance":
+				case "UncloakType":
+				case "TimeOrder":
+				case "ScrollPanelAlign":
+				case "SourceType":
+				case "BlendMode":
+				case "EnterBehaviour":
+				case "TrailType":
+				case "BeamRenderableShape":
+					return ParseEnum(fieldName, value, fieldType);
+				case "BitSet`1":
```

Comment:
Possible typo?

Reply:
Looks like it. Didn't notice any related issues, but no need to leave it.